### PR TITLE
Tmz 3.0.0 iau constallations - closes #1179

### LINKED ISSUE #1179: List of IAU designated constellations
https://en.wikipedia.org/wiki/IAU_designated_constellations
https://de.wikipedia.org/wiki/Liste_der_Sternbilder

### PR DESCRIPTION
**Pull request by t-m-z**

### CODE REVIEW

**Comment by S-Man42** [comment]
Conflicts

**Comment by t-m-z** [comment]
conflicts solved

File: assets/i18n/en.json [hunk 1]
```diff
@@ -2369,6 +2369,106 @@
...
+  "iau_constellation_nasa" : "NASA Abbrevation",
+  "iau_constellation_position" : "Position",
+  "iau_constellation_visibility" : "Visibility",
+  "iau_constellation_area" : "Area",
+  "iau_constellation_star" : "Brightest Star",
+  "iau_constellation_magnitudo" : "Magnitudo",
+  "iau_constellation_andromeda": "the Chained Maiden",
+  "iau_constellation_antlia": "the Air Pump",
+  "iau_constellation_apus": "the Bird of Paradise",
+  "iau_constellation_aquarius": "the Water Bearer",
```
Comment (S-Man42) [inline assets/i18n/en.json:2387]: Please remove the "the"s

Comment (S-Man42) [reply assets/i18n/en.json:2387]: And prepare yourself for adding the correct translations to Crowdin before any of the volounteer translaters simply uses Google Translate :D :D

**Comment by S-Man42** [comment]
Opening the IAU tool:

======== Exception caught by widgets library =======================================================
The following StateError was thrown building IAUConstellations(dirty, dependencies: [_LocalizationsScope-[GlobalKey#927a4]], state: IAUConstellationsState#c640f):
Bad state: No element

The relevant error-causing widget was: 
  IAUConstellations IAUConstellations:file:///E:/workspace/GCWizard/lib/application/registry.dart:1282:25
When the exception was thrown, this was the stack: 
#0      ListBase.firstWhere (dart:collection/list.dart:132:5)
#1      IAUConstellationsState._buildOutput (package:gc_wizard/tools/science_and_technology/astronomy/iau_constellation/widget/iau_constellation.dart:164:34)
#2      IAUConstellationsState.build (package:gc_wizard/tools/science_and_technology/astronomy/iau_constellation/widget/iau_constellation.dart:135:9)
#3      StatefulElement.build (package:flutter/src/widgets/framework.dart:5198:27)
#4      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5086:15)
#5      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#6      Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#7      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5068:5)
#8      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5242:11)
#9      ComponentElement.mount (package:flutter/src/widgets/framework.dart:5062:5)
...     Normal element mounting (112 frames)
#121    Element.inflateWidget (package:flutter/src/widgets/framework.dart:3971:16)
#122    MultiChildRenderObjectElement.inflateWidget (package:flutter/src/widgets/framework.dart:6570:36)
#123    MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6582:32)
...     Normal element mounting (266 frames)
#389    Element.inflateWidget (package:flutter/src/widgets/framework.dart:3971:16)
#390    MultiChildRenderObjectElement.inflateWidget (package:flutter/src/widgets/framework.dart:6570:36)
#391    Element.updateChild (package:flutter/src/widgets/framework.dart:3708:18)
#392    RenderObjectElement.updateChildren (package:flutter/src/widgets/framework.dart:6153:32)
#393    MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6595:17)
#394    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#395    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#396    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#397    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#398    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#399    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#400    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#401    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#402    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#403    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#404    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#405    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#406    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#407    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:107:11)
#408    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#409    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#410    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#411    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#412    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#413    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#414    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#415    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#416    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#417    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:107:11)
#418    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#419    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#420    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#421    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#422    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#423    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#424    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#425    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#426    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#427    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#428    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#429    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6442:14)
#430    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#431    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6442:14)
#432    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#433    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#434    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#435    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#436    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#437    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#438    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#439    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#440    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#441    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#442    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#443    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#444    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#445    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:107:11)
#446    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#447    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6442:14)
#448    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#449    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#450    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#451    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#452    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#453    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#454    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#455    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#456    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#457    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#458    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#459    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#460    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#461    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#462    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#463    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#464    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#465    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#466    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#467    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#468    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#469    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#470    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#471    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#472    StatelessElement.update (package:flutter/src/widgets/framework.dart:5162:5)
#473    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#474    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#475    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#476    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#477    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#478    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#479    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#480    StatelessElement.update (package:flutter/src/widgets/framework.dart:5162:5)
#481    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#482    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#483    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#484    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#485    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#486    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#487    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#488    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#489    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#490    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#491    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#492    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#493    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#494    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#495    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#496    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#497    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#498    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#499    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#500    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#501    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#502    StatelessElement.update (package:flutter/src/widgets/framework.dart:5162:5)
#503    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#504    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#505    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#506    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#507    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#508    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#509    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#510    StatelessElement.update (package:flutter/src/widgets/framework.dart:5162:5)
#511    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#512    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#513    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#514    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#515    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#516    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#517    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#518    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#519    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#520    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6442:14)
#521    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#522    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#523    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#524    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#525    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#526    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#527    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6442:14)
#528    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#529    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#530    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#531    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#532    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:107:11)
#533    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#534    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#535    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#536    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#537    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#538    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#539    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#540    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#541    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#542    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#543    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#544    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#545    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#546    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#547    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#548    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#549    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#550    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#551    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#552    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#553    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6442:14)
#554    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#555    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#556    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#557    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#558    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:107:11)
#559    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#560    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#561    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#562    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#563    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#564    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#565    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#566    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#567    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#568    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#569    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#570    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#571    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#572    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#573    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#574    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#575    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#576    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#577    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#578    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#579    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6442:14)
#580    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#581    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#582    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#583    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#584    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:107:11)
#585    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#586    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#587    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#588    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#589    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#590    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#591    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#592    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#593    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#594    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#595    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#596    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#597    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#598    StatelessElement.update (package:flutter/src/widgets/framework.dart:5162:5)
#599    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#600    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6442:14)
#601    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#602    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#603    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#604    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#605    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:107:11)
#606    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#607    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#608    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#609    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#610    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#611    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#612    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#613    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#614    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#615    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#616    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#617    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#618    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#619    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#620    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#621    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#622    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#623    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#624    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#625    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#626    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#627    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#628    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#629    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#630    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#631    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#632    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#633    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#634    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#635    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#636    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#637    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#638    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#639    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#640    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#641    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#642    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#643    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#644    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#645    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#646    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#647    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#648    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#649    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6442:14)
#650    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#651    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#652    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#653    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#654    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:107:11)
#655    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#656    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#657    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#658    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#659    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#660    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#661    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#662    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#663    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#664    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#665    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#666    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#667    ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)
#668    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#669    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#670    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#671    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#672    StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)
#673    Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)
#674    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)
#675    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)
#676    Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)
#677    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2780:19)
#678    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:903:21)
#679    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:358:5)
#680    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1284:15)
#681    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1214:9)
#682    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1072:5)
#683    _invoke (dart:ui/hooks.dart:142:13)
#684    PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:359:5)
#685    _drawFrame (dart:ui/hooks.dart:112:31)
====================================================================================================
D/EGL_emulation(25239): app_time_stats: avg=43.34ms min=2.03ms max=686.37ms count=18
D/EGL_emulation(25239): app_time_stats: avg=1669.55ms min=1.60ms max=25045.79ms count=22

**Comment by S-Man42** [comment]
grafik

"90° bis 60°" --> The "bis" should be translated or replaced by a dash or something.

**Comment by S-Man42** [comment]
Please add the "GCWDefaultOutput" between input dropdowns and the output. Spacing looks a bit better and it stays consistent to the app

**Comment by S-Man42** [comment]
Hm, the "Sort by" is not quite useful I guess... Is there anybody who says: Oh I want to select a star but first I sort by area, so I can scroll to the 13th brightest star... And then counts the Dropdown to this point? I understand what you are trying to achieve, but maybe you should do something similar to the PSE. The you can chose a table, ordered by "Density". This table shows a ranking number and the name of the element. Then you can tap the element to open its details page. Some should work here as well:

Chose "Order by Brightest star" -> output a ranking table -> Tap the 13th brightest star.

**Comment by S-Man42** [comment]
What is the source of the images?

File: lib/tools/science_and_technology/astronomy/iau_constellation/logic/iau_constellation_data.dart [hunk 2]
```diff
@@ -0,0 +1,987 @@
+// https://en.wikipedia.org/wiki/IAU_designated_constellations
+// https://de.wikipedia.org/wiki/Liste_der_Sternbilder
+// https://www.iau.org/public/themes/constellations/
+
+part of 'package:gc_wizard/tools/science_and_technology/astronomy/iau_constellation/logic/iau_constellation.dart';
+
+enum IAU_CONSTELLATION_SORT { CONSTELLATION, NAME, STAR, AREA, VISIBILIY, MAGNITUDO }
```
Comment (S-Man42) [inline lib/tools/science_and_technology/astronomy/iau_constellation/logic/iau_constellation_data.dart:7]: Are these maps private possible?

File: assets/i18n/en.json (hunk 1 above)
Comment (t-m-z) [reply assets/i18n/en.json:2387]: Yes that's what I am expecting

**Comment by t-m-z** [comment]
> 

https://de.wikipedia.org/wiki/Liste_der_Sternbilder

license will be added. 

A 2nd source could be https://www.iau.org/public/themes/constellations/  with Attribution 4.0 International (CC BY 4.0) 

which is the better source?

**Comment by t-m-z** [comment]
> 

the data is from https://de.wikipedia.org/wiki/Liste_der_Sternbilder

Dabei steht N für die Nord- und S für die Südhalbkugel. Mit N S oder S N gekennzeichnete Sternbilder liegen ziemlich mittig auf dem Himmelsäquator; mit dem Großteil auf der jeweils erstgenannten Halbkugel. Bei N s und S n liegt nur ein sehr kleiner Teil des Sternbildareals auf der mit dem Kleinbuchstaben bezeichneten Halbkugel.

**Comment by t-m-z** [comment]
puih - some heavy homework and refactoring ...

**Comment by S-Man42** [comment]
> which is the better source?
fine for me

**Comment by S-Man42** [comment]
> Dabei steht N für die Nord...

Ok, if it is common Notation, it is fine. Looked for me buggy, but then it is ok

**Comment by S-Man42** [comment]
conflicts.

**Comment by S-Man42** [comment]
grafik
 I have no idea, which one is the star and what is the other column :D Maybe a header line makes sense here.

You can do:
```

outputData = [null, 'My First Column', 'My Second Column'];  // Add First Record with Header Information
outputData.addAll(_buildValueOutputs());   // Add all Data after Headers

--> multiLineColumnedOutput(
   hasHeader: true; // --- Configures the Output to take the first Record as Header
)
```

And maybe you should think about, putting the ordering column first. It's less confusing in my personal opinion.

**Comment by S-Man42** [comment]
grafik

I am not quite sure, but in my feeling it is better, to copy the value of ordering instead of the ... other one :D

Btw: Area: Does it has any unit? Same for Magnitudo (is that a correct English term, sounds Italian or something)?

**Comment by S-Man42** [comment]
Maybe we should call it "Sternbilder" or similar. Does anybody know what "IAU Constellation" mean? Me not...

**Comment by S-Man42** [comment]
Technically greatly done, nonetheless. Only a few usability points...

**Comment by t-m-z** [comment]
Thanks a lot
- Headers included
-  columns switched
- search strings added
- values are copied
- translation checked and corrected
- area unit added

**Comment by S-Man42** [comment]
Still three points, sorry:

1. 
grafik

Use flexValue to reduce width of first column here. (in other categories it's fine). Or maybe it makes sense to add the "name" column here as well. So users see: Oh, "Apus" is "Birds of Paradise" on a quick view.

2. You write "English name". Do you want to keep it for other languages? Or do you want to translate it. In translation it makes no sense for me to write "German name", "Italian name", ... It's simply the name. It only makes sense, when you want to explain a German user, that this is not the name in German but in English.

3. I am still struggling with the tool name. IAU is not very commonly known. I guess nobody would search for IAU if he wants to know about stars and <was heißt Sternbild auf Englisch?>.

**Comment by t-m-z** [comment]
1. tricky but done
2. I have chosen "Official/IAU Name" and "Name"
3. Constellation is the correct word. However a combination of "Stellar Images/Constellations" is fine, too.

**Pull request merged**